source/Calamari.Common/Features/Scripting/Bash/Bootstrap.sh

```diff
@@ -307,6 +307,70 @@ function decrypt_and_parse_variables {
     exec 3<&-
 }
 
+# -----------------------------------------------------------------------------
+# Functions to report Kubernetes Manifests via service message
+#	Accepts 2 arguments:
+#	  string: the namespace
+#	  string: the manifest
+# -----------------------------------------------------------------------------
+function report_kubernetes_manifest 
+{
+  MANIFEST="$1"
+  NAMESPACE="$2"  
+  
+  LINES=()
+
+	# Read lines from file into the array
+	while IFS= read -r LINE; do
+		LINES+=("$LINE")
+	done <<< "$MANIFEST"
+
+  MANIFESTS=()
+  
+  CURRENT=""
+	for LINE in "${LINES[@]}"; do
+		if [ "$LINE" = "---" ]; then
+			MANIFESTS+=("$CURRENT")
+			CURRENT=""
+		else
+			CURRENT="$CURRENT$LINE\n"	
+		fi
+  done
```

Comment:
If someone has a better way to split a variable on a specific set of chars (or even a regex), please let me know, my bash is pretty weak